### PR TITLE
Improve debugger page at narrow widths

### PR DESCRIPTION
**Pull request by DanTup**
Includes various tweaks to support collapsing Octicons buttons (our others all use our own icons). I added a `PButton.octicon` constructor to simplify the code slightly, though so far I've only applied it in the debugger page - if these changes look good I can update the other places to use it too.

Screenshot 2019-03-26 at 5 59 44 pm

Screenshot 2019-03-26 at 5 59 52 pm

Screenshot 2019-03-26 at 6 00 01 pm

Screenshot 2019-03-26 at 6 00 10 pm

### CODE REVIEW

**Comment by kenzieschmoll** [comment]
Do the buttons have tooltips? This will be important when the text is gone.

File: packages/devtools/web/styles.css [hunk 1]
```diff
@@ -867,19 +869,35 @@ span.strong {
...
+    .btn-group.collapsible-1200 .optional-text span:not(.octicon) {
         display: none;
     }
 
     .btn-group.collapsible-1200 .btn .flutter-icon {
         margin-right: 0;
         box-sizing: content-box;
     }
+
+    .btn-group.collapsible-1200 .btn .flutter-icon {
```
Comment (jacob314) [inline packages/devtools/web/styles.css]: Isn't this rule identical to the one right above it?

**Comment by DanTup** [comment]
> Do the buttons have tooltips? This will be important when the text is gone.

Whoops, good spot! They do now.

File: packages/devtools/web/styles.css (hunk 1 above)
Comment (DanTup) [reply packages/devtools/web/styles.css]: Yeah, I wanted to be extra sure it worked 😏

(Gone now)

**Pull request merged**